catalogue_pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/sierra/subjects/SierraOrganisationSubjects.scala

```diff
@@ -0,0 +1,84 @@
+package uk.ac.wellcome.platform.transformer.sierra.transformers.sierra.subjects
+
+import uk.ac.wellcome.models.work.internal._
+import uk.ac.wellcome.platform.transformer.exceptions.TransformerException
+import uk.ac.wellcome.platform.transformer.sierra.source.{
+  SierraBibData,
+  VarField
+}
+import uk.ac.wellcome.platform.transformer.sierra.transformers.sierra.SierraConcepts
+
+trait SierraOrganisationSubjects extends SierraConcepts {
+
+  // Populate wwork:subject
+  //
+  // Use MARC field "610".
+  //
+  // *  Populate the platform "label" with the concatenated values of
+  //    subfields a, b, c, d and e.
+  //
+  // *  Populate "concepts" with a single value:
+  //
+  //    -   Create "label" from subfields a and b
+  //    -   Set "type" to "Organisation"
+  //    -   Use subfield 0 to populate "identifiers", if present.  Note the
+  //        identifierType should be "lc-names".
+  //
+  // https://www.loc.gov/marc/bibliographic/bd610.html
+  //
+  def getSubjectsWithOrganisation(
+    bibData: SierraBibData): List[Subject[MaybeDisplayable[Organisation]]] =
+    getMatchingVarFields(bibData, marcTag = "610").map { varField =>
+      val label =
+        createLabel(varField, subfieldTags = List("a", "b", "c", "d", "e"))
+
+      val organisation = createOrganisation(varField)
+
+      Subject(
+        label = label,
+        concepts = List(organisation)
+      )
+    }
+
+  private def createOrganisation(
+    varField: VarField): MaybeDisplayable[Organisation] = {
+    val label = createLabel(varField, subfieldTags = List("a", "b"))
+    val organisation = Organisation(label = label)
+
+    // @@AWLC: I'm not sure if this can happen in practice -- but we don't have
+    // enough information to build the Organisation, so erroring out here is
+    // the best we can do for now.
+    if (label == "") {
+      throw TransformerException(
+        s"Not enough information to build a label on $varField")
+    }
+
+    getIdentifierSubfieldContents(varField) match {
+      case Seq(identifier) => {
+        val sourceIdentifier = SourceIdentifier(
```

Comment:
Because we don't expect any authority other than library on congress, for person subjects we only extract identifiers if the second indicator is 0. I know the 610 rules don't explictly say that but it feels like they should behave in the same way in this particular case...

Reply:
Or maybe it's worth checking with @silveroliver if that's the case here